js/source/clustered_geojson_worker_source.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var supercluster = require('supercluster');
+var GeoJSONWorkerSource = require('./geojson_worker_base');
+
+var workerSource = Object.create(GeoJSONWorkerSource);
+workerSource.indexData = function (data, params, callback) {
```

Comment:
I see that this method is used to invoke `geojsonvt` elsewhere. Would `createTileFromData` be a good name? I'm still unsure.